### DISCUSSION TITLE
Use hiragana as default on windows 10

### DISCUSSION THREAD

**Discussion by CcMenta** [google/mozc#936] [Q&A]
I use windows 10 and want to learn japanese and because I use a hungarian keyboard any changed what layout should IME use in regedit, so it would use hungarian layout instead of english layout, but it has the consequence of that the ~ is made with AltGr + 1 and the shortcut key to switch between hiragana and "direct input" is Alt + ~, so I can't switch between them.
If there is a way to change the shortcut key for switching between input mods that can be also helpful.

**Comment by niwazukihon** [comment]
You can use this autohotkey script.

```ahk
#Requires AutoHotkey v2.0

#c:: {
    static isJP := 0
    if isJP = 0 {
        Send "{vk16sc1F2}"
        isJP := 1
    }
    else {
        Send "{vk1Asc1F1}"
        isJP := 0
    }
}
```

Pressing `Win+C` switches between hiragana and direct input mode.

By the way, there's also a library [here](https://qiita.com/kenichiro_ayaki/items/d55005df2787da725c6f), but as it says, I tested that its `IME_GetConvMode( )` and `IME_SetConvMode( )` functions doesn't work with Mozc (but works with Microsoft IME).
